### DISCUSSION TITLE
Aggregate Marker Questions

### DISCUSSION THREAD

**Discussion by tmqdeborah** [crystalyragui/MARC2RDA#459] [General]
Here are questions about Aggregate Markers.

=======================================================
Please do not start a new comment to reply to a question; use REPLY instead =======================================================

Aggregate Markers are pattern matches that will (hopefully) reliably identify that a MARC record is describing one of the following types of manifestations:
•	Collection aggregate manifestation
•	Parallel aggregate manifestation
•	Augmentation aggregate manifestation
•	Single expression manifestation (i.e., not an aggregate manifestation)

- [Question](https://github.com/uwlib-cams/MARC2RDA/discussions/459#discussioncomment-10318152): Is it safe to assume an Aggregate manifestation when: No 1XX is present, AND 245$c begins with 'edit'? **Assuming "Yes"**
- [Question](https://github.com/uwlib-cams/MARC2RDA/discussions/459#discussioncomment-10318237): Is it safe to assume an Aggregate manifestation when: 130 is present, AND 245$c begins with 'edit'? **Assuming "Yes"**
- [Question](https://github.com/uwlib-cams/MARC2RDA/discussions/459#discussioncomment-10526223): Is the presence of $k Selections a marker for an collection aggregate? **GD says No**
- [Question](https://github.com/uwlib-cams/MARC2RDA/discussions/459#discussioncomment-10620362): What is the intended use of $i Container of (work) and $i Container of (expression) in a 7XX field? **PCC guidance indicates "Yes"**
- [Question](https://github.com/uwlib-cams/MARC2RDA/discussions/459#discussioncomment-10756574): Are the LCGFT terms "Part songs" and "Part books" indicators of multiple expressions?
- [Question](https://github.com/uwlib-cams/MARC2RDA/discussions/459#discussioncomment-10816754): Are Radio and TV shows Multi-part works or Aggregating works?

**Comment by tmqdeborah** [comment]
### Question: Is it safe to assume an Aggregate manifestation when: No 1XX is present, AND 245$c begins with 'edit'?

If a record has no Agent main entry (100, 110, 111) and has a statement of responsibility (245$c) that begins with 'edit' (e.g., edited by, editor), can we safely assume that the record is describing an aggregate manifestation?

Examples:

- 245 00 $a Technology and the good life? / $c edited by Eric Higgs, Andrew Light, and David Strong. 
- 245 00 $a Advances in distributed and parallel knowledge discovery / $c edited by Hillol Kargupta and Philip Chan.
- 245 00 $a The Valley of Mexico : $b studies in pre-Hispanic ecology and society / $c edited by Eric R. Wolf.
- 245 00 $a John Maynard Keynes, critical assessment. $n Second series / $c edited by John Cunningham Wood.

**Reply by tmqdeborah** [reply]
Seeing no replies, I have proceeded with this assumption.

See the following results from my small sample file: 
[50_CAM_AMTest_MReview.txt](https://drive.google.com/drive/folders/18-wtvXDhWQze9oRsw3tMS3hKtfeah4vo)

**Comment by tmqdeborah** [comment]
### Question: Is it safe to assume an Aggregate manifestation when: 130 is present, AND 245$c begins with 'edit'?

If a record has a uniform title main entry (130) and has a statement of responsibility (245$c) that begins with 'edit' (e.g., edited by, editor), can we safely assume that the record is describing an aggregate manifestation?

Examples:

130 0  $a Gertrude Stein. $l English.
245 10 $a Gertrude Stein : $b in words and pictures : a photobiography / $c edited by Renate Stendhal.

130 0  $a Osnovnye tendenëtìsii soëtìsial§nogo razvitiëiìa Rossii, 1960-1990.
245 10 $a Recent social trends in Russia, 1960-1995 / $c edited by Irene A. Boutenko and Kirill E. Razlogov.

130 0  $a Work (M.I.T. Press)
245 10 $a Work / $c edited by Friederike Sigler.

130 0  $a Anthrozoology (Oxford University Press)
245 10 $a Anthrozoology : $b human-animal interactions in domesticated and wild animals / $c edited by Geoff Hosey and Vicky Melfi.

**Reply by tmqdeborah** [reply]
Seeing no replies, I have proceeded with this assumption.

I found no results in my small sample file, but there will be quite a few in the full file (examples shown above)

**Comment by tmqdeborah** [comment]
### Question: Is the presence of $k Selections a marker for an collection aggregate?

That is, is a collection of 'Selections' from a single work an aggregating work?

@GordonDunsire, if someone has selected and arranged 'selections' from a single work, does that produce an aggregating work? 

For example:
700 12 $a Strouse, Charles. $t Dance a little closer. $k Selections.

Are those selections individual expressions (part expressions) of the single work and, therefore, the selection and arrangement of those expressions constitute an aggregating work?

**Reply by GordonDunsire** [reply]
@tmqdeborah: No.

The original work may not be a whole-part work, and the selections may not be complete part-expressions.

The best RDA can currently do is to say that the Selections work is derived from the original work. Specifically, we can assume it is derived via a representative expression of the original work, which has no 'content' to select from. This goes beyond selections from a language-based/textual work. I can think of 'selections' of an image work, such as villages and towns selected from a medieval cartographic work.

**Reply by tmqdeborah** [reply]
@GordonDunsire: 

I don't understand what you mean by: "Specifically, we can assume it is derived via a representative expression of the original work, which has no 'content' to select from".

You say that a collection of selections from another work is not an aggregating work, but in that case, what are they? Single works? Amalgamated works? Something else?

I agree that the selections are often not **named** parts. They seem to me to be similar to 'excerpts' and 'fragments' and you and I never did finish our analysis of where those types of content fit in the model.

My thinking is this: If an original work is a single part work with a creator responsibility, and someone pulls a **single** excerpt or fragment from that single part work, then the creator of the original work is still responsible for that **single** fragment or excerpt and the excerpt or fragment can be recorded as such. As such, it seems to me that a fragment or excerpt is a kind of unnamed part of an original work.

However, if someone selects **multiple** excerpts or fragments from someone else’s work and arranges them, then I believe that a new work has been created and the responsibility for the new work is not the creator of the original work, but the person who pulled out the pieces and put them together.

This is similar to when someone takes named parts of a work and puts them together; the original creator is responsible for the named parts of the original work but someone else is responsible for putting those parts together, usually with other content, to create another work that is an aggregating work.

When a 240 or 130 contains $k Selections **and** a Conventional Collective Title (CCT) then I think we can agree that is a reliable indicator that a record is describing an aggregating work, e.g.: 
100 1  $a Orlandersmith, Dael.
240 10 $a Plays. $k Selections
245 10 $a Beauty's daughter ; Monster ; The gimmick : $b three plays / $c Dael Orlandersmith.
_This record is describing a collection aggregating work and, as we know, Orlandersmith is not responsible for this aggregating work_

And I think that when a 240 or 130 contains $k Selections it is also a reliable aggregate marker for the following examples, without CCTs:

For example, these 240 do not contain CCT:
100 0  $a Aristotle.
240 10 $a Eudemian ethics. $k Selections. $l English
245 10 $a Aristotle's Eudemian ethics, books I, II, and VIII / $c translated with a commentary by Michael Woods.
_This is a collection of 3 **named parts** (Book I, Book II, and Book VIII) of a multi-part work by Aristotle, but Aristotle is not responsible for this collection aggregating work_

100 1  $a Marjouram, William, $d -1862.
240 10 $a Memorials of Sergeant Marjouram, Royal Artillery, including six years service in New Zealand during the Maori War. $k Selections
245 10 $a Sergeant, sinner, saint, and spy : $b the Taranaki War diary of Sergeant William Marjouram, R.A. / $c edited by Laurie Barber, Garry Clayton, and John Tonkin-Covell.
_Marjouram is responsible for the original work, "Memorials of Sergeant Marjouram, Royal Artillery, including six years service in New Zealand during the Maori War", but he is not responsible for the new work that is a collection of unnamed excerpts from his original work _

100 1  $a Loewe, Frederick, $d 1901-1988.
240 10 $a My fair lady. $k Selections; $o arranged
245 10 $a Music from My fair lady $h [sound recording] / $c Ruby Braff/Dick Hyman.
_This record is not describing the complete work "My far lady" but arrangements (expressions) of selected excerpts, or maybe named parts of that original work._

100 1  $a Grinèev, A. V. $q (Andreæi Val§terovich), $e author.
240 10 $a Alëiìaska pod krylom dvuglavogo orla. $k Selections. $l English
245 10 $a Russian colonization of Alaska : $b preconditions, discovery, and initial development, 1741-1799 / $c Andrei Val'terovich Grinèev ; translated by Richard L. Bland.

100 1  $a Asbj²rnsen, Peter Christen, $d 1812-1885.
240 10 $a Norske folke-eventyr. $k Selections. $l English
245 10 $a East o' the sun and west o' the moon : $b and other fairy tales / $c George Webbe Dasent; illustrated by Marty Nobel.
_This record is describing a collection of selected Norse fairy tales_

100 1  $a Adams, Scott, $d 1957-
240 10 $a Dilbert. $k Selections
245 10 $a You don't need experience if you've got attitude / $c by Scott Adams.
_This record is describing a collection of selected Dilbert comic strips_

100 1  $a Evans, Greg, $d 1947-
240 10 $a Luann. $k Selections
245 10 $a Sometimes, you just have to make your own rules : $b a Luann book / $c by Greg Evans.
300    $a 1 v. (unpaged) : $b ill. ;
500    $a Selections from the author's comic strip: Luann.
_This record is describing a collection of selected Luann comic strips_

100 1  $a Denny, Ebenezer, $d 1761-1822.
240 10 $a Military journal of Major Ebenezer Denny. $k Selections
245 10 $a Denny's vocabulary of Delaware / $c recorded by Ebenezer Denny.
_This record is describing a collection of excerpts from Denny's journal, perhaps related to vocabulary terms_

100 1  $a Nursi, Said, $d 1873-1960.
240 10 $a Risale-i nur. $k Selections. $l English
245 14 $a The rays : $b reflections on Islamic belief, thought, worship, and action / $c Bedièuzzaman Said Nursi ; translated by Hèuseyin Akarsu.

100 1  $a Gardner, Helen, $d 1878-1946.
240 10 $a Art through the ages. $k Selections
245 10 $a Gardner's art through the ages : $b a concise global history / $c Fred S. Kleiner.

100 1  $a Schulz, Charles M. $q (Charles Monroe), $d 1922-2000. $e author.
240 10 $a Peanuts. $k Selections
245 10 $a Good grief, it's your birthday! : $b growing up without growing old / $c [Charles Schulz].
_This record is describing a collection of selected Peanuts comic strips_

100 1  $a Herriman, George, $d 1880-1944, $e artist, $e author.
240 10 $a Krazy Kat. $k Selections
245 14 $a The kat who walked in beauty : $b the panoramic dailies of 1920 / $c by George Herriman ; edited by Derya Ataker.

Here are some '130' examples. Should all Bible works always be considered aggregating works? Should all encyclopedias?
130 0  $a Bible. $k Selections. $l English. $s Authorized. $f 1961.
245 14 $a The bedside Bible : $b an anthology for the quiet hours / $c selected and arranged with introductions and notes by Arthur Stanley.

130 0  $a Bible. $k Selections. $l English. $s English Standard. $f 2006.
245 10 $a He is risen, indeed! / $c paintings by David Erickson.

130 0  $a Encyclopedia of world cultures. $k Selections.
245 10 $a Cultures of the world : $b selections from the ten-volume encyclopedia of world cultures / $c edited by Melvin Ember, Carol R. Ember.

**Reply by GordonDunsire** [reply]
@tmqdeborah: Your question concerns a single (non-aggregating) work from which selections are made.

It is not possible to 'select' anything from a work; the selection must be done from a realization of the work; that is, an expression.

If the expression is not specified, we must assume it is a/the representative expression of the work from which a selection is made.

The new work cannot be an aggregating work. For this to be true, what is selected must be a set of expressions, which can only occur when the source work is an aggregating work. This is ruled out by the basic scenario that the source work is not an aggregating work.

Therefore the new work must be a single, non-aggregating work. The creator of the new work is not, in general, the creator of the source work.

If only one selection is made from the source work, the creator of the source work is not, in general, the creator of the new work; the creator of the source work has no intention that some other agent will make such a selection, and has no control over what is selected. This is the case irrespective of how many selections are in the new work.

The new non-aggregating work is thus a derivation of the source non-aggregating work.

I'm not sure why you are quoting example of aggregating works, or switching from added entry to main entry fields. If the source work is an aggregating work or a whole-part work, the selections in the new work are either whole expressions or fragments of expressions. If whole expressions, the new work is a re-aggregation of a smaller set of expressions than is aggregated in the source work. If fragments of expressions, the above reasoning for the case of a non-aggregating work applies. Either way, the new work is derived from the source work, has a different creator, and its categorization is either aggregating work or non-aggregating work.

**Reply by tmqdeborah** [reply]
@GordonDunsire:

> It is not possible to 'select' anything from a work; the selection must be done from a realization of the work; that is, an expression.

Ok, now I understand what you meant. In future I will try to remember to say “expression/work”.

> The new work cannot be an aggregating work. For this to be true, what is selected must be a set of expressions

Yes, I did realize that an aggregating work is a set of expressions; which is why I was suggesting that selections that are excerpts/fragments from a single expression are ‘**unnamed** parts’ of that expression and could **possibly** be handled in the same way as ‘named parts’ of an expression, i.e., as separate expressions from the whole expression from which they have been pulled. 😊

If this approach works, then when:
- when only **one** excerpt (unnamed part) is selected from a source expression/work, the situation is just the same as when one named part of a multi-part expression/work or one expression/work is selected, i.e., the original creator is still responsible for the selected excerpt expression/work or named part expression/work or whole expression/work, but is not responsible for the resulting aggregating expression/work, IF the excerpt is put together with other content

- when multiple excerpt selections are made from a single expression/work, the situation is just the same as when multiple expression/works are selected, i.e., the original creator is still responsible for their selected expressions/works but is not responsible for the aggregating expression/work that is the result of selecting and arranging the multiple selections, with or without other content.


> I'm not sure why you are quoting example of aggregating works, or switching from added entry to main entry fields.

I sent those mixed examples to show how difficult (impossible) it is going to be to distinguish when $kSelections is being applied to a single expression/work vs. an aggregating expression/work. 

Which means that we will only be able to use $kSelections as an aggregate marker in conjunction with a CCT, e.g.: 
100 1  $a Orlandersmith, Dael.
240 10 $a **Plays**. $k Selections
245 10 $a Beauty's daughter ; Monster ; The gimmick : $b three plays / $c Dael Orlandersmith.

BTW, I retract my 7XX examples (sent previously) because we have already decided that all added entries must be treated as works, with no way of distinguishing between single-part works, multi-part works, and aggregating works. 

The bottom line is that we may not be able to resolve this issue at this time. So, for this project, it appears that we will have to say that **selections from a single work do not result in an aggregating work**.

Which means that all of the examples without CCT (in the comment above) will have to be treated as non-aggregating works, unless there are other aggregate markers in the records for them.

**Reply by GordonDunsire** [reply]
@tmqdeborah: 'Which means that we will only be able to use $kSelections as an aggregate marker in conjunction with a CCT'. I agree!

**Comment by tmqdeborah** [comment]
### Question: What is the intended use of $i Container of (work) and $i Container of (expression) in a 7XX field?

I have found over 6989 tags in 2156 records (in the LC file of 5.6 million) and over 67040 tags in 13345 records (in the UW file of 890K records) where a 7XX field contains $i Container of (work) or $i Container of (expression).

In my sampling of the results log for each file, it appears that the vast majority of the 7XX with the $i repeats a corresponding 7XX without the $i.

700 12$aRilke, Rainer Maria,$d1875-1926.$tWeise von Liebe und Tod des Cornets Christoph Rilke.
700 12$aRilke, Rainer Maria,$d1875-1926.$tWeise von Liebe und Tod des Cornets Christoph Rilke.$lEnglish.
700 12$iContainer of (work):$aRilke, Rainer Maria,$d1875-1926.$tWeise von Liebe und Tod des Cornets Christoph Rilke.


700 12$aKim, Po-ra,$d1981-$tOÌ†je ruÌ†l koÌ†nnoÌ†on onuÌ†l iranuÌ†n segye.
700 1 $aPak, Ch'an-uk,$d1963-$eInterviewee.
700 12$6880-06$iContainer of (work):$aKim, Po-ra,$d1981-$tOÌ†je ruÌ†l koÌ†nnoÌ†on onuÌ†l iranuÌ†n segye.


700 12$aMonteverdi, Claudio,$d1567-1643.$tMissa da cappella fatta sopra il motetto In illo tempore dal Gomberto.
700 12$aVicentino, Nicola,$d1511-approximately 1576.$tMadrigals,$mvoices (5),$nbook 5.$pLaura che il verde lauro.
700 12$aTudino, Cesare,$dapproximately 1530-approximately 1590.$tAmor, i'ho molti e molt'anni pianto.
700 12$aWert, Giaches de,$d1535-1596.$tMadrigals,$nbook 9.$pMia benigna fortuna.
700 12$aMarenzio, Luca,$d1553-1599.$tMadrigals,$mvoices (5),$nbook 9.$pSolo e pensoso i piuÌ€ deserti campi.
700 12$iContainer of (work):$aMonteverdi, Claudio,$d1567-1643.$tMissa da cappella fatta sopra il motetto In illo tempore dal Gomberto.
700 12$iContainer of (work):$aVicentino, Nicola,$d1511-approximately 1576.$tMadrigals,$mvoices (5),$nbook 5.$pLaura che il verde lauro.
700 12$iContainer of (work):$aTudino, Cesare,$dapproximately 1530-approximately 1590.$tAmor, i'ho molti e molt'anni pianto.
700 12$iContainer of (work):$aWert, Giaches de,$d1535-1596.$tMadrigals,$nbook 9.$pMia benigna fortuna.
700 12$iContainer of (work):$aMarenzio, Luca,$d1553-1599.$tMadrigals,$mvoices (5),$nbook 9.$pSolo e pensoso i piuÌ€ deserti campi.


700 12$aSchumann, Robert,$d1810-1856.$tFantasiestuÌˆcke,$mpiano,$nop. 12.
700 12$aSchumann, Robert,$d1810-1856.$tKreisleriana.
700 12$iContainer of (work):$aSchumann, Robert,$d1810-1856.$tFantasiestuÌˆcke,$mpiano,$nop. 12.
700 12$iContainer of (work):$aSchumann, Robert,$d1810-1856.$tKreisleriana.


700 12$aProkofiev, Sergey,$d1891-1953.$tConcertos,$mviolin, orchestra,$nno. 1, op. 19,$rD major.
700 12$aProkofiev, Sergey,$d1891-1953.$tConcertos,$mviolin, orchestra,$nno. 2, op. 63,$rG minor.
700 12$aProkofiev, Sergey,$d1891-1953.$tSonatas,$mviolin,$nop. 115.
700 12$iContainer of (work):$aProkofiev, Sergey,$d1891-1953.$tConcertos,$mviolin, orchestra,$nno. 1, op. 19,$rD major.
700 12$iContainer of (work):$aProkofiev, Sergey,$d1891-1953.$tConcertos,$mviolin, orchestra,$nno. 2, op. 63,$rG minor.
700 12$iContainer of (work):$aProkofiev, Sergey,$d1891-1953.$tSonatas,$mviolin,$nop. 115.


@AdamSchiff, is there a written PCC or other community instruction about adding these particular non-RDA work and expression relationships? Many of the examples are records for musical works, so perhaps Cate might know of such an instruction. 

I was going to use them as aggregate markers, but now I am thinking that we will have to write coding to ignore fields that contain them so that we don't mint duplicate entities for them.

BTW, [we aren't mapping PCC or LC labels for WEMI relationships](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-9822204]), but these would be given the [default relationship](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10075917]) _related work of manifestation_.

**Reply by tmqdeborah** [reply]
I found no instructions in MLA best practices, but did find examples in [Supplements to Best Practices for Music Cataloging. Supplement 3](https://cmc.wp.musiclibraryassoc.org/wp-content/uploads/sites/5/2020/02/Supp_RDA_Best_Practices3.pdf)]: Complete MARC Record Examples Version 1.81 31 December 2019.

These examples do seem to indicate that this relationship is supposed to be used for analytical added entries, i.e., additional embodied works. 

I already look for 7XX Ind2=2 as an aggregate marker, but I will also look for '7XX$iContainer of' when 7XX Ind2 is not 2 (i.e., an error)

**Comment by AdamSchiff** [comment]
It's not completely clear that all of these are aggregates.  Not sure about:


Research trends in solid mechanics : $b report from the United States National Committee on Theoretical and Applied Mechanics / $c editor, George J. Dvorak.

We can't tell from this one whether there are separate works aggregated or not.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Reply by tmqdeborah** [reply]
@AdamSchiff, I finally found the example ("Research trends ... " in the sample file for the first question above, and I agree that it is difficult to tell, from the data provided, whether the record is actually describing an aggregating work. 

All I can say is that **all** of the pattern matches that I am using as 'aggregate markers' are 'blunt instruments' and might pick up some records that describe only one expression rather than multiple expressions. If any of my somewhat cursory examinations of  the result files from my small example set look 'iffy', I then run the same pattern matches on the 5.6 mil file of LC records that I have. If that shows that too many non-aggregating works have been caught (in my estimation), then I either tweak or abandon the marker.

Fortunately, when non-aggregating works are caught by my aggregate markers, the main difference in the transformation will be that a 1XX relationship will be given as a 'contributor [agent] to aggregate' instead of a 'related [agent] of work' relationship.

I am putting all of the reports from my runs on the 15K sample file in this folder: [New 2_AMTest MReview results](https://drive.google.com/drive/folders/18-wtvXDhWQze9oRsw3tMS3hKtfeah4vo).

And the patterns that I am using are all listed in this [Spreadsheet](https://urldefense.com/v3/__https:/drive.google.com/drive/folders/1XHovmbYkegZEqFXPQfkZCI1TPR2lyHO___;!!K-Hz7m0Vt54!mzuxvfRu1YsCee0r4ggAuvm2uaLtsCAa0P6uZvOBIEvZBEbDnK6AYLQZeaqcWeQQDr37TwPqzVu-_JhF7GA$) at the second tab "New 2_AMTest.mrc"

And the Item lists are in this folder: [Aggregate markers Term lists](https://drive.google.com/drive/folders/1XHovmbYkegZEqFXPQfkZCI1TPR2lyHO_)

If anyone examines the results files and find that any of the aggregate marker pattern matches are finding an **unacceptable** percentage of non-aggregating works, **please** let me know and I will either tweak or remove those markers. Thanks!

**Comment by AdamSchiff** [comment]
This repeating of the same access point with and without a relationship designator is an error, which I suspect comes from OCLC synchronization.  I hadn't noticed this happening before, but we should probably alert OCLC to this problem.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Reply by tmqdeborah** [reply]
@AdamSchiff One common feature seems to be a difference in the placement of the $i--at the beginning of the field vs. at the end of the field, e.g.:

010    $a    00714498
700 12 $a Finzi, Gerald, $d 1901-1956. $t New Year music. $i Container of (work):
700 12 $i Container of (work): $a Finzi, Gerald, $d 1901-1956. $t New Year music.

010    $a    00715613 
700 12 $a Brooks, Richard, $d 1942- $t Moby Dick. $k Selections. $i Container of (work):
700 12 $i Container of (work): $a Brooks, Richard, $d 1942- $t Moby Dick. $k Selections.

010    $a    67038882 
700 12 $a Byrd, William, $d 1539 or 1540-1623. $t Psalmes, sonets, and songs. $p Virginella. $i Container of (expression):
700 12 $i Container of (expression): $a Byrd, William, $d 1539 or 1540-1623. $t Psalmes, sonets, and songs. $p Virginella.

Whatever the reason is for this happening, hopefully we will be able to match to on the triple for a minted IRI for the duplicated entity, to avoid creating multiple entity descriptions of the same entity and multiple instances of the same relationship for the same entity.

**Comment by AdamSchiff** [comment]
In answer to your question, PCC policy is spelled out in http://www.loc.gov/aba/pcc/sct/documents/rel-desig-guide-bib.pdf.  See Guidelines 13-14:

The use of relationship designators for resource-to-resource relationships is encouraged.

If a cataloger wishes to indicate a known relationship to a known resource, and the $i relationship information subfield is defined for the MARC 7XX field being used, provide a relationship designator. Do so even if the field coding otherwise already expresses a relationship.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Reply by tmqdeborah** [reply]
Thanks, @AdamSchiff. It is good to have confirmation about PCC's intended use of the "Contributor of ..." relationship.

In the document you reference, I see that Guideline 15: Multiple Relationships, says: 
"Where multiple relationships exist, e.g., an abridged translation, provide separate access points, each
with a single relationship designator in a single $i subfield. Alternatively, identify one relationship as
primary and record that relationship alone."

This will lead to repetition of an access point for the same entity, with different relationships. Which makes it very important for us to be able to match on a minted IRI for the entity, to avoid creating multiple entity descriptions of the same entity.

**Comment by AdamSchiff** [comment]
It would be helpful to have the OCLC number or DLC LCCN of these records so that we could find them and fix them (or report to OCLC).  It's quite difficult to find these records in OCLC using just the fields shown in the examples.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Reply by tmqdeborah** [reply]
I provide LCCNs in the review reports, but didn't think to add them in the examples I copied here. I'll try to remember to add them to all copied examples in future.

**Comment by AdamSchiff** [comment]
Very odd.

Here's what these 7XX fields look like in the corresponding records in OCLC Connexion:

70012ǂi Container of (work): ǂa Finzi, Gerald, ǂd 1901-1956. ǂt Concertos, ǂm clarinet, string orchestra, ǂn op. 31, ǂr C minor.
70012ǂi Container of (work): ǂa Finzi, Gerald, ǂd 1901-1956. ǂt Romance, ǂm string orchestra, ǂn op. 11, ǂr E♭ major.
70012ǂi Container of (work): ǂa Finzi, Gerald, ǂd 1901-1956. ǂt New Year music.
70012ǂi Container of (work): ǂa Finzi, Gerald, ǂd 1901-1956. ǂt Dies natalis.

70012ǂi Container of (work): ǂa Brooks, Richard, ǂd 1942- ǂt Moby Dick. ǂk Selections.
70012ǂi Container of (work): ǂa Austin, Elizabeth R., ǂd 1938- ǂt Symphonies, ǂn no. 1.
70012ǂi Container of (work): ǂa Brooks, Richard, ǂd 1942- ǂt Landscape... with grace.
70012ǂi Container of (work): ǂa DiArta Angeli, F. ǂt Sta pestá.

They are perfectly fine in OCLC.  So I looked in the LC online catalog, and the MARC looks the same as above:

  *   70012 |i Container of (work): |a Finzi, Gerald, |d 1901-1956. |t Concertos, |m clarinet, string orchestra, |n op. 31, |r C minor.
  *   70012 |i Container of (work): |a Finzi, Gerald, |d 1901-1956. |t Romance, |m string orchestra, |n op. 11, |r E♭ major.
  *   70012 |i Container of (work): |a Finzi, Gerald, |d 1901-1956. |t New Year music.
  *   70012 |i Container of (work): |a Finzi, Gerald, |d 1901-1956. |t Dies natalis.

  *   70012 |i Container of (work): |a Brooks, Richard, |d 1942- |t Moby Dick. |k Selections.
  *   70012 |i Container of (work): |a Austin, Elizabeth R., |d 1938- |t Wilderness symphony.
  *   70012 |i Container of (work): |a Brooks, Richard, |d 1942- |t Landscape... with grace.
  *   70012 |i Container of (work): |a DiArta Angeli, F. |t Sta pestá.


LCCN 67038882 is not in OCLC (!!), although this is what it looks like in the LC catalog viewed online:

  *   7001_ |a Yonge, Nicholas, |d -1619, |e editor.
  *   70012 |i Container of (expression): |a Byrd, William, |d 1539 or 1540-1623. |t Psalmes, sonets, and songs. |p Virginella.

The formatting looks perfectly fine, but this is a work access point, not an expression one!

Bottom line: I'm not sure where your duplicated fields are coming from!  They don't seem to be in the source data from LC and the relationship designators are displaying in front of the access point, as the PCC guidelines prescribe.

  *
 Adam
  *
Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***

________________________________

**Reply by tmqdeborah** [reply]
@AdamSchiff, I think the answer to this puzzle is that my file is from April 2022 and perhaps sometime around then a global fix was run to copy a 700Ind2=2 field and move the $i from the end to the beginning of the field. Then, before another fix could be run to delete the fields with the $i at the end, I got the copy of the file.

And, yes, as you noticed in the Yonge example, the "Container of ... " relationship is not correct. This seems to have happened often enough in my samples for those relationship labels to be undependable for distinguishing between when a work entity or an expression entity is being described by the field.

**Comment by tmqdeborah** [comment]
### Question: Are the LCGFT terms "Part songs" and "Part books" indicators of multiple expressions?###

The MLA RDA Metadata Guidance [Introduction](https://musiclibraryassociation.github.io/mla-rda-mg/Content/Intro/intro.html)  [LRM and RDA Concepts](https://musiclibraryassociation.github.io/mla-rda-mg/Content/LRM/lrm_and_rda_concepts.html)    Musical Works says: "An extremely common case of parallel aggregates in music is a score issued with a set of parts: the same work expressed in two different formats of notated music and manifested in multiple units. "

I take this to mean that the following are good aggregate markers (Scores with 'part' indicators):
- AND 000/06=c||d [Case=Y], AND 300$a=part [Case=N] 
- AND 000/06=c||d [Case=Y], AND 300$e=part [Case=N] 
- AND 000/06=c||d [Case=Y], AND 655$a Parts (Music) [Case=Y]

However, I am not so sure about these LCGFT terms:
- AND 000/06=c||d [Case=Y], AND 655$a Part songs [Case=Y]
- AND 000/06=c||d [Case=Y], AND 655$a Part books [Case=Y]

@AdamSchiff or other Music people, please advise, using the GitHub Reply button :-)

**Comment by tmqdeborah** [comment]
### Question: Are Radio and TV shows Multi-part works or Aggregating works?

1. Are Radio and TV programs such as documentaries, game shows, talk shows, etc., aggregating works? 
    - I think they are.
3. Are short TV series that tell a single story in 2 or more episodes, multi-part works? 
    - I think they are.
5. Are multi-season TV series that tell a single story each season, e.g., "[Vera](https://en.wikipedia.org/wiki/Vera_(TV_series))" or "[Inspector Morse](https://en.wikipedia.org/wiki/Inspector_Morse_(TV_series))", or "[Wallander](https://en.wikipedia.org/wiki/Wallander_(British_TV_series))" aggregating works (the season) with a plan to aggregate multi-part works (the stories told in the episodes)?  
    - I think they are. 
    - But they could be long running narratives (multi-part works) centered around a single character or group of characters. 
    - Whichever is true, would also apply to a book series, e.g., "Cleeves, Ann. Vera Stanhope series."
6. What about soap operas (e.g., "[As the world turns](https://en.wikipedia.org/wiki/As_the_World_Turns)", "[Coronation Street](https://en.wikipedia.org/wiki/Coronation_Street)) or long running shows like "Grey's Anatomy"? 
    - They have been described as a "[https://en.wikipedia.org/wiki/Soap_opera](form of television that works with a continuous open narrative. Each [episode](https://en.wikipedia.org/wiki/Episode) ends with a promise that the storyline is to be continued in another episode)"

If we can pin these questions down, I might be able to use some LCGFT terms as aggregate markers.
